autoload/openbrowser/github.vim

```diff
@@ -294,24 +294,30 @@ function! s:parse_github_remote_url(github_host) abort
   " - git@github.com:tyru/open-github-browser.vim
   " - ssh://git@github.com/tyru/open-github-browser.vim
   let ssh_re_fmt = 'git@%s[:/]\([^/]\+\)/\([^/]\+\)\s'
+  let ssh2_re_fmt = '\s%s[:/]\([^/]\+\)/\([^/]\+\)\s'
```

Comment:
Is `\s` necessary?

Reply:
This is for:

```
$ git remote -v
origin  github.com:tyru/open-browser-github.vim (fetch)
origin  github.com:tyru/open-browser-github.vim (push)
```

and the purpose was not to match something like `another-github.com:tyru/open-browser-github.vim`. 

However, I don't have such an example as of now, so I'm okay with removing it too. Then we could merge it to `ssh_re_fmt` because it would match `git@github.com:tyru/open-github-browser.vim` as well. Do you prefer that?

Reply:
> and the purpose was not to match something like another-github.com:tyru/open-browser-github.vim.

I got it.
The change looks good to me!